src/content/0.15.0/docs/start/anywhere.md

```diff
@@ -7,7 +7,7 @@ There are two options for running Fluree:
 
 ### Download
 
-To run Fluree, you need Java Server 8 (Java 1.8) or above. To verify your version of Java, you can type `java -version` in the terminal.
+To run Fluree, you need Java 11 or above. To verify your version of Java, you can type `java -version` in the terminal.
```

Comment:
Actually, v0.15 & v0.16 were tested/running under Java 8.  
You need to create a copy of this file for v1.0